### ISSUE TITLE
Error: Tags is undefined or not passed as a String

### ISSUE THREAD

**Issue by Jolg42** [logdna/nodejs#38]
When using the example from the README, I get this error `Error: Tags is undefined or not passed as a String`
```
// Add tags in array or comma-separated string format:
options.tags = ['logging', 'nodejs', 'logdna'];
```

It works fine with the alternative, as a string with commas.
Maybe you should remove the array method from the README or fix it?

**Comment by olearycrew** [comment]
+1

**Comment by respectus** [comment]
Should be resolved in release 3.0.0, thanks for the feedback!
